helm/block-producer/templates/block-producer.yaml

```diff
@@ -330,6 +335,21 @@ spec:
             path: keyfile.json
       {{- end }}
 ---
+{{- if $.Values.coda.persistentStorage }}
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: mina-pv-claim
+  labels:
+    app: mina
+spec:
+  accessModes:
```

Comment:
Actually probably want vars for the *access mode* and *capacity request* to ensure binding based on what's available.